File: ZhigaoWeather/app/src/java/com/weather/zhigao/model/CityTable.java
package com.weather.zhigao.model;

public class CityTable {

    public static final String TAB_NAME = "table_city";
    public static final String COL_LOC = "location";
    public static final String COL_COND = "cond_txt";
    public static final String COL_TMP_MIN = "tmp_min";
    public static final String COL_TMP_MAX = "tmp_max";
    public static final String CREATE_TABLE = "create table " + TAB_NAME + " ("
            + COL_LOC + " text primary key,"
            + COL_COND + " text,"
            + COL_TMP_MIN + " text,"
            + COL_TMP_MAX + " text"
            + ")";
}

File: ZhigaoWeather/app/src/main/java/com/weather/zhigao/model/CityTable.kt
package com.weather.zhigao.model

object CityTable {

    val TAB_NAME = "table_city"
    val COL_LOC = "location"
    val COL_COND = "cond_txt"
    val COL_TMP_MIN = "tmp_min"
    val COL_TMP_MAX = "tmp_max"
    val CREATE_TABLE = ("create table " + TAB_NAME + " ("
            + COL_LOC + " text primary key,"
            + COL_COND + " text,"
            + COL_TMP_MIN + " text,"
            + COL_TMP_MAX + " text"
            + ")")
}
